src/App.tsx

```diff
@@ -1,16 +1,74 @@
 import React from 'react';
 import './App.scss';
 
-class App extends React.Component {
-  addOne = () => {};
+type State = {
+  count: number,
+};
 
-  add100 = () => {};
+class App extends React.Component<{}, State> {
+  state: State = {
+    count: 0,
+  };
 
-  increase = () => {};
+  addOne = () => {
+    this.setState((currentState) => ({
+      count: currentState.count + 1,
+    }));
+  };
+
+  add100 = () => {
+    this.setState((current) => ({
+      count: current.count + 100,
+    }));
+  };
+
+  increase = () => {
+    this.addOne();
+
+    if (this.state.count % 5 === 0) {
+      this.add100();
+    }
+  };
 
   render() {
+    const { count } = this.state;
+
     return (
-      <h1>Count: 0</h1>
+      <div
+        className="counter"
+      >
```

Comment:
In one line it looks better
```suggestion
      <div className="counter">
```